ccm_web/server/src/lti/lti.service.ts

```diff
@@ -1,11 +1,13 @@
-import type { Express, Request, Response } from 'express'
-import { BeforeApplicationShutdown, Injectable } from '@nestjs/common'
+import { Express, Request, Response } from 'express'
+import { BeforeApplicationShutdown, HttpStatus, Injectable } from '@nestjs/common'
 import { ConfigService } from '@nestjs/config'
 import { IdToken, Provider as LTIProvider } from 'ltijs'
 import Database from 'ltijs-sequelize'
 
 import baseLogger from '../logger'
 import { DatabaseConfig, LTIConfig } from '../config'
+import { UserService } from '../user/user.service'
+import { User } from '../user/user.model'
```

Comment:
Not sure if `User` is used in this module anymore.

Reply:
it is on line 70, as I am checking if record is User Object or not
```
if (!(record instanceof User)) {
        logger.error(`something went wrong in creating user: ${loginId}`)
        return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json('The Launch of application failed, please try to refresh the page.')
      }
```

Reply:
Oh duh, I had removed that check in my `try/catch` code, so I got confused. Please disregard.